### PR TITLE
Updates to Amazon index

### PR DESCRIPTION
**Pull request by Melvin-CnC**
My first updated page.

## Purpose of this pull request

This pull request (PR) ...updates to index.

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by kimsauce** [comment]
@Melvin-CnC , were you able to preview these changes successfully in your local build (`yarn start`)? It's throwing an error that should've come up prior to pushing your PR.

File: docs/integrations/amazon-aws/index.md [hunk 1]
```diff
@@ -11,7 +11,328 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+        </div>
+        <div className="box box46 card">
+          <div className="container">
+          <img src={useBaseUrl('img/integrations/amazon-aws/waf.png')} alt="Thumbnail icon" width="70"/>
+          <h3><a href="/docs/integrations/amazon-aws">AWS Foundations Benchmark App</a></h3>
+          <p>A guide to the Sumo Logic app for AWS Threat Intel.</p>
+          </div>
+        </div>                                                                                                                
+    </div>
+  </div>
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/index.md:338]:
```suggestion
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Hi @Melvin-CnC , great job overall! Thank you for your hard work. Some edits:

- Most of the URLs are linking back to themselves, rather than the specific apps. For instance, the GitLab URL says `<a href="/docs/integrations/app-development/">GitLab</a>`, but it should be `<a href="/docs/integrations/app-development/gitlab">GitLab</a>`
- When you're previewing your build, make sure the logos and text are visually aligned (height-wise). I've gone in and applied most of the fixes, but there may be others.

cc: @stacykor

File: docs/integrations/amazon-aws/index.md [hunk 2]
```diff
@@ -12,7 +12,325 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
-import DocCardList from '@theme/DocCardList';
-import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
-
-<DocCardList items={useCurrentSidebarCategory().items}/>
+<div className="box-wrapper" markdown="1">
+<div className="box smallbox1 card">
+  <div className="container">
+  <img src={useBaseUrl('/../../../img/integrations/amazon-aws/aws-aurora-logo.png')} alt="Thumbnail icon" width="50"/>
+  <h4><a href="/docs/integrations/amazon-aws/">Amazon Aurora MySQL ULM</a></h4>
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/index.md:19]: @Melvin-CnC , can you please finish the URLs for this page? 

For example, this URL in this particular line would be /docs/integrations/amazon-aws/**aurora-mysql-ulm**

**Review by kimsauce — CHANGES_REQUESTED** [review]
@Melvin-CnC , can you please finish the URLs?

For next time, you can push one page at a time (one page per PR).

File: docs/integrations/amazon-aws/index.md (hunk 2 above)
Comment (Melvin-CnC) [reply docs/integrations/amazon-aws/index.md:19]: Sure @kimsauce, I'll make the changes. Thought, I'd already made them.

**Comment by kimsauce** [comment]
Hi @Melvin-CnC , thank you for your hard work on this! I'm going to push all of your changes to production except the one you had in progress (`/docs/integrations/security-threat-detection`).

**Pull request merged**